input/sample_targetlist_for_cals.csv

```diff
@@ -0,0 +1,9 @@
+# AR1 mosaic NGC641
```

Comment:
This is one of the test files and I seem to have committed a copy of it as input.
Must have been accidental, I removed this one since it is not needed and should not be here.